tools/Makefile.am

```diff
@@ -3,7 +3,8 @@ noinst_PROGRAMS = \
 	cgc \
 	hex2float \
 	fp20 \
-	fx10
+	fx10 \
+	replay
```

Comment:
I think this is in the wrong commit

Reply:
Yes, that's what usually happens if I don't test each commit before pushing :)